src/decode.rs

```diff
@@ -1442,14 +1472,7 @@ unsafe fn decode_b(
     }
 
     let ts = &mut *t.ts;
-    let f = &mut *t.f;
-    let frame_hdr = &***f.frame_hdr.as_ref().unwrap();
```

Comment:
Why was this removed in favor of the `f.frame_hdr()` method?  The method requires an `.unwrap()` and `Arc` deref on every access, and also hides this cost.  I checked and there don't appear to be any borrowck issues keeping the old way.